components/observatorium-api-tls-secret.libsonnet

```diff
@@ -0,0 +1,26 @@
+{
+  local api_tls_secret = self,
+
+  config:: {
+    namespace: error 'must provide namespace',
+  },
+  // WARNING: Never use this example secret in production!
+  secret: {
+    apiVersion: 'v1',
+    kind: 'Secret',
+    metadata: {
+      name: 'observatorium-api-tls-secret',
+      namespace: api_tls_secret.config.namespace,
+    },
+    data: {
+      'ca.pem': 'LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJkVENDQVJxZ0F3SUJBZ0lVRUNEUzQ3Z3ZoUEFWSm5vNTJlOFluemhONlZRd0NnWUlLb1pJemowRUF3SXcKR0RFV01CUUdBMVVFQXhNTmIySnpaWEoyWVhSdmNtbDFiVEFlRncweU1EQTBNVFl4TVRRek1EQmFGdzB5TlRBMApNVFV4TVRRek1EQmFNQmd4RmpBVUJnTlZCQU1URFc5aWMyVnlkbUYwYjNKcGRXMHdXVEFUQmdjcWhrak9QUUlCCkJnZ3Foa2pPUFFNQkJ3TkNBQVQwWDJxR2Y4Z0FKTlRJeWFDc290QXVoY3BweTBIa1NuY0JmdG1aaWpqWUNsZW0KN0JPd2I1a1RScnhaeVJxUU9Kdll0RmV0R1E3T2RjdUJOaVpXb05vem8wSXdRREFPQmdOVkhROEJBZjhFQkFNQwpBUVl3RHdZRFZSMFRBUUgvQkFVd0F3RUIvekFkQmdOVkhRNEVGZ1FVMHUzTm83YnNFekxtTzNYd0h2OUN3S21EClVNY3dDZ1lJS29aSXpqMEVBd0lEU1FBd1JnSWhBUFFsbS8wQWdoSkJEYmxtZHdkRE81V2xEbHZiUFovaE1yc2IKMFh3SHF0aHFBaUVBcXAzaTgvYmJyMnVFZzlYS1RMbE16Sit2OFdHSFIyc0wwYVNqUk9zeG9tUT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=',
```

Comment:
we should pass these in as parameters in the `config`, and you can use `stringData` so this doesn't have to be base64 encoded